pallets/automation-time/src/tests.rs

```diff
@@ -2387,17 +2470,33 @@ fn missed_tasks_removes_completed_tasks() {
 
 		assert_eq!(AutomationTime::get_task_queue().len(), 0);
 		assert_eq!(AutomationTime::get_missed_queue().len(), 0);
+
+		let mut condition: BTreeMap<String, String> = BTreeMap::new();
+		condition.insert(String::from("type"), String::from("time"));
+		condition.insert(String::from("timestamp"), format!("{}", LAST_BLOCK_TIME));
+
 		assert_eq!(
 			events(),
 			[
+				RuntimeEvent::AutomationTime(crate::Event::TaskTriggered {
+					who: owner.clone(),
+					task_id: task_id01.clone(),
+					condition: condition.clone(),
+					encoded_call: vec![],
+					abort_errors: vec![],
+				}),
 				RuntimeEvent::AutomationTime(crate::Event::Notify { message: message_one }),
+				RuntimeEvent::AutomationTime(crate::Event::TaskExecuted {
+					who: owner.clone(),
+					task_id: task_id01.clone(),
+				}),
```

Comment:
If `TaskMissed` fires, the task is missed. There shouldn’t be `TaskTriggered` and `TaskExecuted`.

If it is the 2nd execution that is missed, add comments here to mark the start of the second task execution and explain the logic.